package.json

```diff
@@ -57,8 +64,8 @@
     "premanifest": "cp -R src/js/ dist/",
     "manifest": "npx @custom-elements-manifest/analyzer analyze --config scripts/custom-elements-manifest.config.js",
     "build:types": "tsc",
-    "build:esm": "esbuild \"src/js/**/*.js\" --target=es2019 --format=esm --outdir=dist",
-    "build:cjs": "esbuild \"src/js/**/*.js\" --target=es2019 --format=cjs --outdir=dist/cjs",
+    "build:esm": "esbuild \"src/js/**/*.js\" src/js/react/**/* --target=es2019 --format=esm --outdir=dist",
```

Comment:
These add the react-specific modules to the build process for CJS & ESM (but not for our bundled version).